__tests__/taskRequests/taskRequest.test.js

```diff
@@ -84,28 +96,59 @@ describe('Task Requests', () => {
         const modal = await page.$('.filter-modal');
         const filterHead = await page.$('.filter-head');
         const filterContainer = await page.$('.filters-container');
-
         expect(filterHead).toBeTruthy();
         expect(filterContainer).toBeTruthy();
-
         await page.click('#filter-button');
         expect(modal).not.toBeNull();
         expect(
           await modal.evaluate((el) => el.classList.contains('hidden')),
         ).toBe(false);
-
         await page.mouse.click(20, 20);
         expect(
           await modal.evaluate((el) => el.classList.contains('hidden')),
         ).toBe(true);
       });
+
+      it('checks if PENDING is checked by default', async () => {
+        const filterButton = await page.$('#filter-button');
+        await filterButton.click();
+        await page.waitForSelector('.filter-modal');
+        const activeFilter = await page.$('input[value="PENDING"]');
+        const currentState = await activeFilter.getProperty('checked');
+        const isChecked = await currentState.jsonValue();
+        expect(isChecked).toBe(true);
+      });
+
+      it('Selecting filters and clicking on apply should filter task request list', async () => {
+        await page.click('#filter-button');
+        await page.click('input[value="PENDING"]');
+        await page.click('input[value="APPROVED"]');
+        await page.click('#apply-filter-button');
+        await page.waitForNetworkIdle();
+        const cardsList = await page.$$('.taskRequest__card');
+        expect(cardsList).not.toBeNull();
+        expect(cardsList.length).toBeGreaterThanOrEqual(0);
```

Comment:
can we also check if the length before and after filter are not same so that we can be confident that filter is working as expected.

Reply:
Done.